syntax = "proto3";

message Vote{
    string vote_id = 1;
    uint64 timestamp = 2;
    string id_voter = 3;

    string id_election = 4;
    repeated string id_option = 5;
}

message VoteContainer {
    repeated Vote entries = 1;
}
